grails-app/domain/org/pih/warehouse/inventory/CycleCount.groovy

```diff
@@ -113,6 +113,10 @@ class CycleCount {
         return cycleCountItems.findAll{ it.countIndex == countIndex}
```

Comment:
if you wanted, you could refactor this method to do `return getItemsOfSpecificCount(maxCountIndex)`